src/patternfly/components/Toolbar/toolbar.scss

```diff
@@ -395,6 +395,10 @@ $pf-v5-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2
   &:last-child {
     --#{$toolbar}--spacer: 0;
   }
+
+  .#{$panel} {
+    --#{$panel}--MinWidth: 100%;
+  }
```

Comment:
Can you point link to the examples where this is used? I'm wondering if we need to scope this style further so it doesn't do anything we don't intend.

Reply:
Ping - didn't see a reply here. Similar to the menu toggle comment, I'm curious if this is fixing a known bug and if this might do something we don't want it to since it has a broad scope. If it's a bug fix/enhancement, can we move it into another issue?